lib/test_prof/before_all/adapters/active_record.rb

```diff
@@ -44,8 +44,14 @@ def setup_fixtures(test_object)
       # might lead to leaking connections
       config.before(:begin) do
         next unless ::ActiveRecord::Base.connection.pool.respond_to?(:lock_thread=)
+        @orig_lock_thread = ::ActiveRecord::Base.connection.pool.instance_variable_get("@lock_thread")
```

Comment:
Can we be a bit more defensive here and use a more unique name for the instance variable? (Or even some esoteric one like [here](https://github.com/test-prof/test-prof/blob/e47c9ee3de9d3a2dad0822bb4c9bd93259938210/lib/test_prof/recipes/rspec/let_it_be.rb#L78)?

Reply:
> Or even some esoteric one like [here](https://github.com/test-prof/test-prof/blob/e47c9ee3de9d3a2dad0822bb4c9bd93259938210/lib/test_prof/recipes/rspec/let_it_be.rb#L78)?

wow, that's great 😸